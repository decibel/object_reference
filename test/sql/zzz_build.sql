\set ECHO none

-- Loads deps, but not extension itself
\i test/pgxntool/setup.sql

CREATE EXTENSION IF NOT EXISTS count_nulls;
CREATE EXTENSION IF NOT EXISTS cat_tools;

CREATE SCHEMA object_reference;
\i sql/object_reference.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
