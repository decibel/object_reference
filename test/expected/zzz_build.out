\set ECHO none
This extension must be loaded via CREATE EXTENSION object_reference;
You really, REALLY do NOT want to try and load this via psql!!!
psql:test/temp_load.not_sql:166: WARNING:  I promise you will be sorry if you try to use this as anything other than an extension!






psql:test/temp_load.not_sql:489: WARNING:  I promise you will be sorry if you try to use this as anything other than an extension!









psql:test/temp_load.not_sql:596: WARNING:  I promise you will be sorry if you try to use this as anything other than an extension!

psql:test/temp_load.not_sql:603: WARNING:  I promise you will be sorry if you try to use this as anything other than an extension!













Loaded OK!
# TRANSACTION INTENTIONALLY LEFT OPEN!
